samples/drawing/drawing.cpp

```diff
@@ -822,7 +822,7 @@ void MyCanvas::DrawText(wxDC& dc)
     dc.DrawRotatedText( wxT("That is text"), 20, 10, -45 );
 
     // use wxSWISS_FONT and not wxNORMAL_FONT as the latter can't be rotated
-    // under Win9x (it is not TrueType)
+    // under MSW (it is not TrueType)
```

Comment:
Personally I don't really know. Is wxNORMAL_FONT a TrueType under WinXP for example? Would it be better if it read "under some Windows versions" ?